tools/testing.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import inspect
+
+
+def run_in_pyodide(_function=None, standalone=False, packages=[]):
+    """
+    This decorator can be called in two ways --- with arguments and without
+    arguments. If it is called without arguments, then the `_function` kwarg
+    catches the function the decorator is applied to. Otherewise, standalone
+    and packages are the actual arguments to the decorator.
+
+    See docs/testing.md for details on how to use this
```

Comment:
Please document parameters in the docstring still and ideally add type annotations.